scripts/update_version.py

```
@@ -0,0 +1,582 @@
+# TypeScript SDK Changelog
+
+
+# Lines starting with # are comments and will be ignored
+"""
+        
+        with tempfile.NamedTemporaryFile(mode='w', suffix='.md', delete=False) as f:
+            f.write(template)
+            temp_file = f.name
+        
+        # Open in vim
+        subprocess.call(['vim', temp_file])
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)